mesonbuild/modules/wayland.py

```diff
@@ -98,12 +100,16 @@ def scan_xml(self, state: ModuleState, args: T.Tuple[T.List[FileOrString]], kwar
             targets.append(code)
 
             for side in sides:
+                command = [self.scanner_bin, f'{side}-header', '@INPUT@', '@OUTPUT@']
+                if kwargs['core_only']:
+                    command.append('-c')
```

Comment:
I think you should use the long argument instead of the short one. Makes reading the code a little easier.

Reply:
Oh good point, did not notice there is `--include-core-only`.